### ISSUE TITLE
 Idea: "Chill Camera" mode to give the game a snappier feeling

### ISSUE THREAD

**Issue by Tsavoritel** [shishkabob27/CardWars#31]
The idea is to have less camera movement during gameplay. I think it could make things feel snappier and maybe even speed up the game a bit. Right now, the camera moves a lot, and it kinda unnecessarily adds to the game feeling a bit slow.
Thought it'd be cool to have an option to keep the camera more static.

I'm thinking a simple toggle in the settings for "Reduced Camera Motion" or "Chill Camera" mode.
It could be a nice way to cater to players who prefer a quicker pace.

Got a bit excited and started coding this. Got a basic version working on my pc.
It's not super polished as I'm not the best with coding, but happy to share if you guys are interested.

If it sounds good, I'll tidy up what I've got and open a pull request.

**Comment by Sabersabersaber123** [comment]
i agree, or just a option that makes the game go faster when placing cards n stuff

**Comment by bpdrej** [comment]
> The idea is to have less camera movement during gameplay. I think it could make things feel snappier and maybe even speed up the game a bit. Right now, the camera moves a lot, and it kinda unnecessarily adds to the game feeling a bit slow. Thought it'd be cool to have an option to keep the camera more static.
> 
> I'm thinking a simple toggle in the settings for "Reduced Camera Motion" or "Chill Camera" mode. It could be a nice way to cater to players who prefer a quicker pace.
> 
> Got a bit excited and started coding this. Got a basic version working on my pc. It's not super polished as I'm not the best with coding, but happy to share if you guys are interested.
> 
> If it sounds good, I'll tidy up what I've got and open a pull request.

That sounds great! Have you tried testing it in Unity Editor?

**Comment by Tsavoritel** [comment]
Of course! As a novice coder, I had to test it a few times before getting it to work properly. It also seems to function well when built. I'll definitely do some more tests though. Also, I plan to remove one or two animations because keeping the camera static in certain scenarios feels awkward and, ironically, less snappy. After I get these changes, I'll go ahead and submit the PR!
